tests/test_bonded.py

```diff
@@ -167,6 +168,58 @@ def test_flat_bottom_bond(self, n_particles=64, n_bonds=35, dim=3):
             np.testing.assert_array_equal(test_du_dx, test_du_dx_rev)
             np.testing.assert_array_equal(test_du_dp, test_du_dp_rev)
 
+    def test_log_flat_bottom_bond(self, n_particles=64, n_bonds=35, dim=3):
+        """Randomly connect pairs of particles, then validate the resulting LogFlatBottomBond force"""
+        np.random.seed(2022)
+
+        # TODO(deboggle) : reduce code duplication between HarmonicBond and LogFlatBottomBond
+        box = np.eye(3) * 100
+        x = self.get_random_coords(n_particles, dim)
+
+        atom_idxs = np.arange(n_particles)
+
+        k = np.random.rand(n_bonds) * 10  # k not too large to avoid exp(-inf)
```

Comment:
Is this important? How big can k be?

Applying the assumption that these terms will keep the central particle nearly fixed (consequence of crowded environment), we have been we have been considering values of k not constrained by this consideration, and have typically been setting k a few orders of magnitude larger than this.

Also, worth noting that exp(-inf) for all r < r_max, for any k > 0, so there is no "safe" value of k in this regard...

Reply:
Turns out this test still passes for ```k=np.random.rand(n_bonds)*1000``` - just wanted to conservative here. Will adjust to k =

Reply:
Not for this PR, but we will need to adjust the limit for local MD if we think there is an upper bound on the stability of this (https://github.com/proteneer/timemachine/blob/master/timemachine/cpp/src/local_md_utils.cu#L159)

Reply:
+1 - good call